XenAdmin/Core/Updates.cs

```diff
@@ -442,6 +442,9 @@ public static List<XenServerPatch> RecommendedPatchesForHost(Host host)
                 var minimumPatches = new List<XenServerPatch>();

```

Comment:
These two lines can be compacted:
`var minimumPatches = serverVersions[0].MinimalPatches;`